### PR TITLE
add test for ctr and upd test for lambda

### PR DESCRIPTION
**Pull request by ryanishe**

### CODE REVIEW

File: tests/functional/tests/test_observe_observables.py [hunk 1]
```diff
@@ -1,15 +1,74 @@
...
 
 
-def test_positive_relay_observe_observables_domain(relay_api):
-    """ Test relay observe observables api mock for domain in Graph Security
+def test_positive_enrich_observe_observables_sha256(module_headers):
+    """Perform testing for enrich observe observables endpoint for sha256 in
+    Graph Security module
 
-    ID: CCTRI-354-4c96ad24-5a16-449b-9120-0af449d546c6
+    CCTRI-354-38e92acf-45e2-4ff6-b5f4-c7e7f4e20f2d
```
Comment (oshtaier) [inline tests/functional/tests/test_observe_observables.py]: any reason to remove `ID:` tag?

File: tests/functional/tests/test_observe_observables.py [hunk 2]
```diff
@@ -188,95 +270,42 @@ def test_positive_relay_observe_observables_hostname(relay_api):
         assert expected_observable[key] == sighting[key]
 
 
-def test_positive_relay_observe_observables_sha256(relay_api):
```
Comment (oshtaier) [inline tests/functional/tests/test_observe_observables.py]: any reason to remove test completely?

File: tests/functional/tests/test_observe_observables.py [hunk 3]
```diff
@@ -188,95 +270,42 @@ def test_positive_relay_observe_observables_hostname(relay_api):
...
+        1. Check that data in response body contains expected information
+            from Microsoft Graph Security module
 
     Importance: Critical
     """
     observable = 'C:\\Windows\\SYSTEM32\\ntdll.dll'
     observable_type = 'file_path'
 
     expected_observable = {
-        'description': (
```
Comment (oshtaier) [inline tests/functional/tests/test_observe_observables.py]: any reason to remove these parameters?
Are you able to test that type at all?

File: tests/functional/tests/test_relay_api.py [hunk 4]
```diff
@@ -31,3 +31,125 @@ def test_positive_relay_api(relay_endpoint, relay_api):
     # Check status
     assert relay_api.__getattribute__(relay_endpoint)(
         payload=observables).status_code == 200
+
+
+def test_positive_lambda_relay_observe_observables_domain(relay_api):
```
Comment (oshtaier) [inline tests/functional/tests/test_relay_api.py]: I think it is better to rename file to `test_lambda_relay_api` and remove `lambda_relay` from all the tests

File: tests/functional/tests/test_observe_observables.py (hunk 1 above)
Comment (ryanishe) [reply tests/functional/tests/test_observe_observables.py]: returned ID: tag

File: tests/functional/tests/test_observe_observables.py (hunk 2 above)
Comment (ryanishe) [reply tests/functional/tests/test_observe_observables.py]: this test is not completely removed, it is now the first

File: tests/functional/tests/test_relay_api.py (hunk 4 above)
Comment (ryanishe) [reply tests/functional/tests/test_relay_api.py]: updated

File: tests/functional/tests/test_observe_observables.py (hunk 3 above)
Comment (ryanishe) [reply tests/functional/tests/test_observe_observables.py]: I will add these parameter

**Pull request merged**